test/app/datasources/car-makes-chained-endpoint.json

```diff
@@ -0,0 +1,35 @@
+{
+	"datasource": {
+		"key": "car-makes-chained-endpoint",
+		"name": "Makes datasource",
+		"source": {
+			"type": "remote",
+			"protocol": "http",
+			"host": "127.0.0.1",
+			"port": "3000",
+			"endpoint": "1.0/makes/{name}"
```

Comment:
What is going on with the spacing here? 😱

Reply:
fixed :)